examples/specs/waterfall_chart.vl.json

```diff
@@ -21,7 +21,7 @@
   "width": 800,
   "height": 450,
   "transform": [
-    {"window": [{"op": "sum", "field": "amount", "as": "sum"}]},
+    {"joinaggregate": [{"op": "sum", "field": "amount", "as": "sum"}]},
```

Comment:
@kanitw We use joinaggregate for the window where it makes sense. The name became a bit unwieldy and I don't think we need to list every feature in the filename for this complex example. I was thinking that we maybe want a separate section for complex/custom designs at the bottom of the examples page.

Reply:
You can keep the name if you have strong preference, but window doesn't make sense here.   See SVG.